admin_manual/appliance/howto-update-owncloud.rst

```diff
@@ -162,6 +163,14 @@ This is because the earlier versions of ownCloud are installed with a different
 More specifically, the versions of the ownCloud app, prior to 10, have a version suffix in the name. 
 For example the ownCloud 8.2 app is named ``owncloud82``.
 
+For owncloud82 users: During the upgrade to the owncloud App, the user files will be moved to the new docker data directory `/var/lib/univention-appcenter/apps/owncloud/data/files`. Essentially, the following the command will be executed:
```

Comment:
``upgrade to the owncloud App`` --> ``upgrade of the owncloud App``

Reply:
well, it's tricky. Because you are upgrading from owncloud82 to owncloud.

So you are in a way upgrading to something different.

Reply:
I'd write it a little differently:

> For ownCloud 8.2 users: during the ownCloud App upgrade, user files will be moved to the new Docker data directory, ``/var/lib/univention-appcenter/apps/owncloud/data/files``.  Essentially, the following the command will be executed:

Reply:
Q: the following the command? 
or is it better to say the following command

Reply:
If this is in relation to my comment, then "the following command".